gravitee-apim-console-webui/src/management/api/api-traffic-v4/webhook-logs-details/webhook-logs-details.component.ts

```diff
@@ -234,36 +347,21 @@ export class WebhookLogsDetailsComponent implements OnInit {
     }
   }
 
-  private buildConnectionFailure(log: WebhookLog): ConnectionFailureDetails | null {
-    if (log.status !== 0) {
-      return null;
-    }
-
-    return {
-      errorKey: log.errorKey ?? null,
-      componentType: log.errorComponentType ?? null,
-      componentName: log.errorComponentName ?? null,
-      message: log.message ?? null,
-      diagnostics: log.warnings ?? [],
-      lastError: log.additionalMetrics?.['string_webhook_last-error'] ?? null,
-    };
-  }
-
   private formatPayloadSize(log: WebhookLog): string {
-    const size = log.additionalMetrics?.['int_webhook_resp-body-size'];
+    const size = log.additionalMetrics[RESP_BODY_SIZE_KEY];
     if (size === null || size === undefined) {
       return '—';
     }
-    if (size >= 1024) {
+    if (isNumber(size) && size >= 1024) {
       return `${(size / 1024).toFixed(1)} KB`;
     }
     return `${size} B`;
   }
 
-  private formatDate(value: string): string {
+  private formatDateTime(value: string | number): string {
     const date = new Date(value);
     if (Number.isNaN(date.getTime())) {
-      return value;
+      return isNumber(value) ? String(value) : value;
```

Comment:
Could you instead do : 
```suggestion
      return String(value);
```
? Because this way you will always be sure the returned value is a string. Or is it not something you want?

Reply:
Yeah not sure what was happening in the brain here. Fixed [fcbd571](https://github.com/gravitee-io/gravitee-api-management/pull/14303/commits/fcbd571882cefbb6a80b68352efc6ac594d43cd6)